### DISCUSSION TITLE
lazer does not interact well with my tablet

### DISCUSSION THREAD

**Discussion by overflowskies** [ppy/osu#21013] [Q&A]
Attempting to enable the "Tablet" option does not allow me to change sensitivity (like I can on osu!stable) - I've read that changing the aspect ratio may help, but the sensitivity still feels far too low.

If I disable the Tablet option, and use Mouse, when I enable "High Precision Mouse" to change my sensitivity, a section of my tablet (around the middle, considering full area) becomes unresponsive (this does not happen on stable / outside of osu), and when I don't enable it, naturally I can't change my sensitivity (so it's too low). Enabling or disabling Touch input did nothing.

Not sure if videos / screenshots can help, but I can definitely take some in case it's needed.

For reference, this has been going on ever since i installed lazer. I've installed all the updates, but nothing. I'm currently running Windows 10, and my tablet is the CTL-4100 (with default Wacom drivers).

**Comment by jamesbt365** [comment]
You don't touch sensitivity with tablet because it messes with absolute input. You change the area. If you don't know how to get the area you want its current area / sens = actual area.

**Reply by overflowskies** [reply]
What value represents the "current area"? I tried experimenting with aspect ratio but I assume it's not the right choice.

**Reply by jamesbt365** [reply]
Whatever your area currently is that you want to convert. If your currently using wacom coordinates you can convert it using this math: http://opentabletdriver.net/Wiki/FAQ/General#area-conversion
